en/user_guide/keyword_management.rst

```diff
@@ -0,0 +1,12 @@
+Keyword management
+==================
+
+Open Orchestra allows to attach keyword to content or media. They describe the content and allows to manage it.
+In the keyword panel, you can create or destroy keyword.
+
+Usage
+-----
+
+When you fill meta for content or media, already created keyword can be choice or you can create new on the keyword field.
+
+For the beta, their only use on front office in the "content list block" and "media list by keyword" to filter displaying content from a list.
```

Comment:
do you mean "their only use is to" or "they are only used to" ? You have to choose ;)